pyscf/ci/cisd.py

```diff
@@ -730,7 +795,7 @@ def __call__(self, mol_or_geom, ci0=None, **kwargs):
                 # FIXME: Whether to use the initial guess from last step?
                 # If root flips, large errors may be found in the solutions
                 ci0 = self.ci
-            self.kernel(ci0, **kwargs)[0]
+            _ = self.kernel(ci0, **kwargs)[0]
```

Comment:
I think, I'll reverse this change for example to be in line with other parts of the pyscf code.